### ISSUE TITLE
why not merge with pl_bolts ? 

### ISSUE THREAD

**Issue by k-nar** [ayasyrev/pl_utils#12]
have you considered adding ImagenetteDataModule  to  

https://github.com/PyTorchLightning/pytorch-lightning-bolts/tree/master/pl_bolts/datamodules

I find Imagenette quite useful for fast iterative experiments which incidentally  is pl_bolts main goal

**Comment by ayasyrev** [comment]
Yes. I'm going to do this, have some idea to refactor and need to document it.
